androidApp/src/main/java/dev/johnoreilly/starwars/androidApp/MainActivity.kt

```diff
@@ -76,7 +92,16 @@ fun MainLayout() {
     }
 }
 
-
+@Composable
+private fun rememberStarWarsRepository(): StarWarsRepository {
+    val context = LocalContext.current.applicationContext
+    val sqlNormalizedCacheFactory = remember { SqlNormalizedCacheFactory(context, "swapi.db") }
```

Comment:
It'd be nice I think if we could do this in one place in shared code.  Is there reason why we wouldn't use this for all clients in the same way?

Reply:
Only the constructor is dependent on the `actual` type.  So 

`SqlNormalizedCacheFactory(context, "swapi.db")`

Context may not be there universally.

My readins is that you need different constructor calls.  https://www.apollographql.com/docs/kotlin/caching/normalized-cache/#sqlite-cache

Reply:
ah, ok....would probably have to use expect/actual then to manage that in shared code.  I've run in to the `context` dependency like this in other repos....in ones that use Koin for example I include this when initialising Koin on Android so that the other `actual` implementation on Android can then just use `get()` to depend on that.  I can follow up after this MR and set that up (and can test on iOS then as well).

```
androidContext(this@GalwayBusApplication)
```